### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by snize**
Currently can't get pre-installed content which described.

### CODE REVIEW

**Comment by dawehner** [comment]
Thank you for creating a pull request!

File: README.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ cd <DESTINATION>/web
 Once your site is running locally, you might want to use Curl to examine the pre-installed content:
 
 ```bash
-curl --header 'Accept: application/vnd.api+json' http://127.0.0.1:8888/api/recipes
+curl --header 'Accept: application/vnd.api+json' http://127.0.0.1:8888/api/node/recipe
```
Comment (dawehner) [inline README.md:41]: I'm a bit confused to be honest. At least in my [contenta elm](https://github.com/contentacms/contenta_jsonapi__elm) consumer I'm using the previous path.

Can you maybe provide some more context how you've used contenta?

Comment (snize) [reply README.md:41]: I just follow [Installation for Building Your Own Site](https://github.com/contentacms/contenta_jsonapi#installation-for-building-your-own-site) and install contenta with sqlite. 
But can't get data with `curl --header 'Accept: application/vnd.api+json' http://127.0.0.1:8888/api/recipes`. I found small difference between README.md and `/admin/api`, So I try `curl --header 'Accept: application/vnd.api+json' http://127.0.0.1:8888/api/node/recipe` 

contenta_json_api

Here's my [composer.lock](https://gist.github.com/snize/2d01069157017100510db6766d6b3c85).

List modules

```
../bin/drush pml
 Package              Name                                                         Type    Status         Version
 Authentication       Simple OAuth (simple_oauth)                                  Module  Enabled        8.x-2.0-rc3
 Authentication       Simple OAuth Extras (simple_oauth_extras)                    Module  Enabled        8.x-2.0-rc3
 Core                 Actions (action)                                             Module  Not installed  8.3.7
 Core                 Activity Tracker (tracker)                                   Module  Not installed  8.3.7
 Core                 Aggregator (aggregator)                                      Module  Not installed  8.3.7
 Core                 Automated Cron (automated_cron)                              Module  Not installed  8.3.7
 Core                 Ban (ban)                                                    Module  Not installed  8.3.7
 Core                 BigPipe (big_pipe)                                           Module  Not installed  8.3.7
 Core                 Block (block)                                                Module  Enabled        8.3.7
 Core                 Book (book)                                                  Module  Not installed  8.3.7
 Core                 Breakpoint (breakpoint)                                      Module  Enabled        8.3.7
 Core                 CKEditor (ckeditor)                                          Module  Enabled        8.3.7
 Core                 Color (color)                                                Module  Not installed  8.3.7
 Core                 Comment (comment)                                            Module  Enabled        8.3.7
 Core                 Configuration Manager (config)                               Module  Enabled        8.3.7
 Core                 Contact (contact)                                            Module  Not installed  8.3.7
 Core                 Contextual Links (contextual)                                Module  Not installed  8.3.7
 Core                 Custom Block (block_content)                                 Module  Not installed  8.3.7
 Core                 Custom Menu Links (menu_link_content)                        Module  Enabled        8.3.7
 Core                 Database Logging (dblog)                                     Module  Not installed  8.3.7
 Core                 Field (field)                                                Module  Enabled        8.3.7
 Core                 Field UI (field_ui)                                          Module  Enabled        8.3.7
 Core                 Filter (filter)                                              Module  Enabled        8.3.7
 Core                 Forum (forum)                                                Module  Not installed  8.3.7
 Core                 Help (help)                                                  Module  Enabled        8.3.7
 Core                 History (history)                                            Module  Not installed  8.3.7
 Core                 Internal Dynamic Page Cache (dynamic_page_cache)             Module  Enabled        8.3.7
 Core                 Internal Page Cache (page_cache)                             Module  Enabled        8.3.7
 Core                 Menu UI (menu_ui)                                            Module  Enabled        8.3.7
 Core                 Node (node)                                                  Module  Enabled        8.3.7
 Core                 Path (path)                                                  Module  Enabled        8.3.7
 Core                 Quick Edit (quickedit)                                       Module  Not installed  8.3.7
 Core                 RDF (rdf)                                                    Module  Not installed  8.3.7
 Core                 Responsive Image (responsive_image)                          Module  Not installed  8.3.7
 Core                 Search (search)                                              Module  Not installed  8.3.7
 Core                 Shortcut (shortcut)                                          Module  Not installed  8.3.7
 Core                 Statistics (statistics)                                      Module  Not installed  8.3.7
 Core                 Syslog (syslog)                                              Module  Enabled        8.3.7
 Core                 System (system)                                              Module  Enabled        8.3.7
 Core                 Taxonomy (taxonomy)                                          Module  Enabled        8.3.7
 Core                 Testing (simpletest)                                         Module  Not installed  8.3.7
 Core                 Text Editor (editor)                                         Module  Enabled        8.3.7
 Core                 Toolbar (toolbar)                                            Module  Enabled        8.3.7
 Core                 Tour (tour)                                                  Module  Not installed  8.3.7
 Core                 Update Manager (update)                                      Module  Enabled        8.3.7
 Core                 User (user)                                                  Module  Enabled        8.3.7
 Core                 Views (views)                                                Module  Enabled        8.3.7
 Core                 Views UI (views_ui)                                          Module  Not installed  8.3.7
 Core (Experimental)  Content Moderation (content_moderation)                      Module  Not installed  8.3.7
 Core (Experimental)  Datetime Range (datetime_range)                              Module  Enabled        8.3.7
 Core (Experimental)  Field Layout (field_layout)                                  Module  Not installed  8.3.7
 Core (Experimental)  Inline Form Errors (inline_form_errors)                      Module  Not installed  8.3.7
 Core (Experimental)  Layout Discovery (layout_discovery)                          Module  Not installed  8.3.7
 Core (Experimental)  Migrate (migrate)                                            Module  Enabled        8.3.7
 Core (Experimental)  Migrate Drupal (migrate_drupal)                              Module  Not installed  8.3.7
 Core (Experimental)  Migrate Drupal UI (migrate_drupal_ui)                        Module  Not installed  8.3.7
 Core (Experimental)  Place Blocks (block_place)                                   Module  Not installed  8.3.7
 Core (Experimental)  Settings Tray (outside_in)                                   Module  Not installed  8.3.7
 Core (Experimental)  Workflows (workflows)                                        Module  Not installed  8.3.7
 Field types          Datetime (datetime)                                          Module  Enabled        8.3.7
 Field types          File (file)                                                  Module  Enabled        8.3.7
 Field types          Image (image)                                                Module  Enabled        8.3.7
 Field types          Link (link)                                                  Module  Enabled        8.3.7
 Field types          Options (options)                                            Module  Enabled        8.3.7
 Field types          Telephone (telephone)                                        Module  Enabled        8.3.7
 Field types          Text (text)                                                  Module  Enabled        8.3.7
 Media                dropzonejs (dropzonejs)                                      Module  Enabled        8.x-1.0-alpha7
 Media                DropzoneJS entity browser widget (dropzonejs_eb_widget)      Module  Enabled        8.x-1.0-alpha7
 Media                Entity Browser (entity_browser)                              Module  Enabled        8.x-1.1
 Media                Entity Browser example (entity_browser_example)              Module  Not installed  8.x-1.1
 Media                Entity Browser IEF (entity_browser_entity_form)              Module  Not installed  8.x-1.1
 Media                Media entity (media_entity)                                  Module  Enabled        8.x-1.6
 Media                Media Entity Browser (media_entity_browser)                  Module  Enabled        8.x-1.0-beta3
 Media                Media entity image (media_entity_image)                      Module  Enabled        8.x-1.2
 Multilingual         Configuration Translation (config_translation)               Module  Not installed  8.3.7
 Multilingual         Content Translation (content_translation)                    Module  Not installed  8.3.7
 Multilingual         Interface Translation (locale)                               Module  Not installed  8.3.7
 Multilingual         Language (language)                                          Module  Enabled        8.3.7
 OpenAPI              OpenAPI (openapi)                                            Module  Enabled        8.x-1.0-alpha1
 OpenAPI              OpenAPI Documentation using Swagger UI (openapi_swagger_ui)  Module  Not installed  8.x-1.0-alpha1
 OpenAPI              OpenAPI Redoc Documentation (openapi_redoc)                  Module  Enabled        8.x-1.0-alpha1
 Other                Contenta specific enhancements (contenta_enhancements)       Module  Enabled
 Other                Entity (entity)                                              Module  Enabled        8.x-1.0-alpha4
 Other                Fieldable Path (fieldable_path)                              Module  Enabled        8.x-1.0-rc4
 Other                Recipes Magazin (recipes_magazin)                            Module  Enabled
 Other                Recipes Magazin Contenta (recipes_magazin_contenta)          Module  Enabled
 Other                Remote stream wrapper (remote_stream_wrapper)                Module  Enabled        8.x-1.0
 Other                snail (snail)                                                Module  Not installed
 Web services         HAL (hal)                                                    Module  Enabled        8.3.7
 Web services         HTTP Basic Authentication (basic_auth)                       Module  Not installed  8.3.7
 Web services         JSON API (jsonapi)                                           Module  Enabled        8.x-1.2
 Web services         JSON API Extras (jsonapi_extras)                             Module  Enabled
 Web services         RESTful Web Services (rest)                                  Module  Enabled        8.3.7
 Web services         Schemata (schemata)                                          Module  Enabled        8.x-1.0-alpha2
 Web services         Schemata in JSON Schema (schemata_json_schema)               Module  Enabled        8.x-1.0-alpha2
 Web services         Serialization (serialization)                                Module  Enabled        8.3.7
 Web services         Subrequests (subrequests)                                    Module  Enabled        8.x-1.0-beta2
 Core                 Bartik (bartik)                                              Theme   Disabled       8.3.7
 Core                 Seven (seven)                                                Theme   Disabled       8.3.7
 Core                 Stark (stark)                                                Theme   Disabled       8.3.7
 Other                Material Admin (material_admin)                              Theme   Enabled
```

Comment (dawehner) [reply README.md:41]: This is essentially the same as https://github.com/contentacms/contenta_jsonapi/issues/160
Thank you for reporting!

**Pull request closed without merging**